docs/operator.md

```diff
@@ -83,22 +84,108 @@ spec:
 EOF
 ```
 
-Create a catalog source for the operator to appear in OperatorHub.
+Having the ClusterDeployment CRD installed is a prerequisite.
```

Comment:
Wonder if it may be worth having all these steps in a script. How are we testing the operator in CI? would put the steps below in a script help with that task?

Reply:
So that is still a TODO for the CI to test the operator. I think @djzager was taking point on that. But for installing it in the CI should be added to the step registry or dev scripts image.